### DISCUSSION TITLE
2 Trojans found in Security Onion Installation .

### DISCUSSION THREAD

**Discussion by Augustin1231** [Security-Onion-Solutions/securityonion#9771] [Unsupported Versions]
Hi Guys !

Downloaded Security Onion and my Anti Virus detected 2 trojans in the installation files . I have read the other posts about the false positives but the names don't correspond . This is why I decided to post here and ask you guys about it as I am not sure what to do . Should I ignore it ? 

Thank you ! 

Gen:Variant.Tedy.260678
XLM.Trojan.Abracadabra.36.gen
Generic.Bat.Downloader.1.71FC1D66

**Comment by presianbg** [comment]
Hi,

If you have downloaded the official .iso, you may be 100% it's clean.
This might be your AV is detecting a suricata or sigma rule detection rules as malicious, because they are trying to match a malicious content / paylods.

What AV are you using ? Are you scanning the whole .iso file ? 

Cheers,
PY

**Comment by Augustin1231** [comment]
I am using Bitdefender Total Security , and I scanned the entire system .

**Comment by cm-ops** [comment]
Is your alert from the following path - `SecurityOnion\agrules\strelka\yara`? If so, see https://docs.securityonion.net/en/latest/download.html second warning on the page.
